tests/e2e/core-tests/specs/merchant/wp-admin-order-searching.test.js

```diff
@@ -53,79 +97,82 @@ const runOrderSearchingTest = () => {
-			await searchForOrder('addr 1', orderId, 'John Doe');
+			await searchForOrder(customerBilling.address_1, orderId, searchString);
 		})
 
 		it('can search for order by billing second address', async () => {
-			await searchForOrder('addr 2', orderId, 'John Doe');
+			await searchForOrder(customerBilling.address_2, orderId, searchString);
 		})
 
 		it('can search for order by billing city name', async () => {
-			await searchForOrder('San Francisco', orderId, 'John Doe');
+			await searchForOrder(customerBilling.city, orderId, searchString);
 		})
 
 		it('can search for order by billing post code', async () => {
-			await searchForOrder('94107', orderId, 'John Doe');
+			await searchForOrder(customerBilling.postcode, orderId, searchString);
 		})
 
 		it('can search for order by billing email', async () => {
-			await searchForOrder('john.doe@example.com', orderId, 'John Doe');
+			await searchForOrder(customerBilling.email, orderId, searchString);
 		})
 
 		it('can search for order by billing phone', async () => {
-			await searchForOrder('123456789', orderId, 'John Doe');
+			await searchForOrder(customerBilling.phone, orderId, searchString);
 		})
 
 		it('can search for order by billing state', async () => {
-			await searchForOrder('CA', orderId, 'John Doe');
+			await searchForOrder(customerBilling.state, orderId, searchString);
 		})
 
 		it('can search for order by shipping first name', async () => {
-			await searchForOrder('Tim', orderId, 'John Doe');
+			await searchForOrder(customerShipping.first_name, orderId, searchString);
 		})
 
 		it('can search for order by shipping last name', async () => {
-			await searchForOrder('Clark', orderId, 'John Doe');
+			await searchForOrder(customerShipping.last_name, orderId, searchString);
 		})
 
 		it('can search for order by shipping first address', async () => {
-			await searchForOrder('Oxford Ave', orderId, 'John Doe');
+			await searchForOrder(customerShipping.address_1, orderId, searchString);
 		})
 
 		it('can search for order by shipping second address', async () => {
-			await searchForOrder('Linwood Ave', orderId, 'John Doe');
+			await searchForOrder(customerShipping.address_2, orderId, searchString);
 		})
 
 		it('can search for order by shipping city name', async () => {
-			await searchForOrder('Buffalo', orderId, 'John Doe');
+			await searchForOrder(customerShipping.city, orderId, searchString);
 		})
 
 		it('can search for order by shipping postcode name', async () => {
-			await searchForOrder('14201', orderId, 'John Doe');
+			await searchForOrder(customerShipping.postcode, orderId, searchString);
 		})
 
-		it('can search for order by shipping state name', async () => {
-			await searchForOrder('CA', orderId, 'John Doe');
+		/**
+		 * shipping state is abbreviated. This test passes if billing and shipping state are the same
+		 */
+		it.skip('can search for order by shipping state name', async () => {
```

Comment:
Do you feel we should put in a follow-up task for this?

Reply:
> Do you feel we should put in a follow-up task for this?

With the data I used, it would do a case insensitive search for `NY`. I'm not sure there is much value in searching for something so easily matched. But, it would be good to release a version of the package where it shows as skipped before we remove it. We probably should discuss how we want to handle this type of situation so that we can be consistent.

Reply:
That makes sense--let's go with that for now.

> We probably should discuss how we want to handle this type of situation so that we can be consistent.

That's a good idea!